src/routes/__tests__/DisciplineSelectionScreen.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import { RouteProp } from '@react-navigation/native'
+import { fireEvent, waitFor } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+
+import { useLoadDisciplines } from '../../hooks/useLoadDisciplines'
+import { RoutesParams } from '../../navigation/NavigationTypes'
+import createNavigationMock from '../../testing/createNavigationPropMock'
+import { getReturnOf } from '../../testing/helper'
+import { mockDisciplines } from '../../testing/mockDiscipline'
+import render from '../../testing/render'
+import DisciplineSelectionScreen from '../DisciplineSelectionScreen'
+
+jest.mock('@react-navigation/native')
+jest.mock('../../hooks/useLoadDisciplines')
+
+describe('DisciplineSelectionScreen', () => {
+  const navigation = createNavigationMock<'DisciplineSelection'>()
+  const parentDiscipline = {
+    id: 0,
+    title: 'Parent Discipline',
+    description: 'Description0',
+    icon: 'none',
+    numberOfChildren: 1,
+    isLeaf: false,
+    parentTitle: null,
+    needsTrainingSetEndpoint: false,
+    leafDisciplines: [10, 11],
+  }
+  const getRoute = (): RouteProp<RoutesParams, 'DisciplineSelection'> => ({
+    key: 'key-1',
+    name: 'DisciplineSelection',
+    params: {
+      discipline: parentDiscipline,
+    },
+  })
```

Comment:
No need for a function:
```suggestion
  const route = {
    key: 'key-1',
    name: 'DisciplineSelection',
    params: {
      discipline: parentDiscipline,
    },
  }
```